tests/compare_result.py

```diff
@@ -41,7 +41,10 @@ def _inner(fout_name, suffix, adpt_num_iter,suffix_ref=None,precision=1E-10):
             E_titles, data_energy, data, data_smooth = read_energyresult_dat(filename)
             path_filename_ref = os.path.join(rootdir, 'reference', filename_ref)
             E_titles_ref, data_energy_ref, data_ref, data_smooth_ref = read_energyresult_dat(path_filename_ref)
-
+            if precision == None:
+                precision = abs(sum(sum(data_smooth_ref))/float(np.prod(np.shape(data_smooth_ref))*1E12) ) 
```

Comment:
Can be written as `np.average(abs(data_smooth_ref)) / 1E12`.

Also, is there a reason to use `data_smooth_ref` instead of `data_ref`?

Reply:
Thanks!

Using data_smooth_ref because I am afraid the peaks from no_smooth data will interfere with the estimate of the average.